src/info.rs

```diff
@@ -39,7 +41,19 @@ impl std::fmt::Display for Info {
             Some(&c) => c,
             None => Color::White,
         };
+        if !self.disable_fields.git_info{
+            let username = if self.git_username == "" {
+                String::from("")
+            }else{
+                String::from(format!("{}:",self.git_username))          
```

Comment:
`format!` already returns type `String`, so you don't need to call `String::from` here. It's kind of like calling `String::from(String::from("stuff"))`.

Reply:
Oh, I didn't know that!
I've removed that redundancy.